e2e/test/scenarios/admin-2/tenants.cy.spec.js

```diff
@@ -243,18 +243,19 @@ describe("Tenants - management", () => {
     cy.findByRole("link", { name: /Tenants/ }).click();
     cy.findByTestId("admin-content-table").should("contain.text", "1");
 
-    cy.findByLabelText("Deactivated").click();
+    cy.findByRole("tab", { name: /Deactivated/ }).click();
     cy.findByTestId("admin-content-table")
       .findByRole("button", { name: /ellipsis/ })
       .click();
     H.popover().findByText("Reactivate tenant").click();
     H.modal().button("Reactivate").click();
-    cy.findByTestId("admin-panel").should(
-      "contain.text",
-      "No matching tenants found.",
+
+    cy.log(
+      "after reactivating the last deactivated tenant, tabs should disappear and show all active tenants",
     );
```

Comment:
According to the Figma design, we should not show tabs if there are no deactivated tenants, so this goes back to showing every active tenants.